client/src/App.tsx

```diff
@@ -44,7 +45,9 @@ const App = (): ReactElement => {
 
   return (
     <Fragment>
-      <RootRoutes isSyncingInProgress={isSyncingInProgress || isFetchingCypressHelpers} />
+      <Layout isSyncingInProgress={isSyncingInProgress || isFetchingCypressHelpers}>
+        <RootRoutes />
+      </Layout>
```

Comment:
In delegation flow, we show views that are outside of the app's layout. I sense this change can break it, can't it?

Reply:
What views do you have in mind? In delegation flow we use only modals.